client/checkout/api/index.js

```diff
@@ -50,6 +50,7 @@ export default class WCPayAPI {
 		if ( ! this.stripe ) {
 			this.stripe = new Stripe( publishableKey, {
 				stripeAccount: accountId,
+				betas: [ 'payment_element_beta_1' ],
```

Comment:
Maybe we should only enable the betas if UPE is enabled since this could affect all stripe code.

Reply:
Good idea! I just added this [config value](https://github.com/Automattic/woocommerce-payments/blob/1e748f957dd221d66cf40aec2b83a9d7ea310fee/client/checkout/api/index.js#L52) to trigger the beta version of Stripe, which gets set in the main payment gateway over [here](https://github.com/Automattic/woocommerce-payments/blob/1e748f957dd221d66cf40aec2b83a9d7ea310fee/includes/class-wc-payment-gateway-wcpay.php#L505).